[AGENTRUN-739] Document agent site changing to fqdn (#31903)
DataDog/documentation: content/en/agent/configuration/network.md

- `apt.datadoghq.com`

## Destinations
<div class="alert alert-warning">

> Starting with version 7.67.0, the Agent converts Datadog sites to fully qualified domain names (ie. it adds a dot at the end of the domain) in order to reduce the number of DNS queries.

Reviewer: ```suggestion
Starting with version 7.67.0, the Agent converts Datadog sites to fully qualified domain names (by adding a dot at the end of the domain) to reduce the number of DNS queries.
```

> For example it sends APM payloads to <code>trace.agent.datadoghq.com.</code>.<br>

Reviewer: ```suggestion
For example, it sends APM payloads to <code>trace.agent.datadoghq.com.</code>.<br>
```

> This behavior can be disabled in version 7.72.0 or more recent by setting <code>convert_dd_site_fqdn.enabled</code> to <code>false</code> in the configuration, or with the environment variable <code>DD_CONVERT_DD_SITE_FQDN_ENABLED=false</code>.

Reviewer: ```suggestion
This behavior can be disabled in version 7.72.0 and later by setting <code>convert_dd_site_fqdn.enabled</code> to <code>false</code> in the configuration, or with the environment variable <code>DD_CONVERT_DD_SITE_FQDN_ENABLED=false</code>.
```
